.github/workflows/release.yml

```diff
@@ -62,10 +70,11 @@ jobs:
       dylib_artifact: libtiledbgenomicsdb.dylib.${{ github.ref_name }}
 
   test:
-    needs: [release-jar]
+    needs: [build-and-push-mac-dylib, release-jar]
```

Comment:
I have not understood how this `TAG_MESSAGE` is being used. Shouldn't `release-jar` already be dependent on `build-and-push-mac-dylib`? Do we have to explicitly call out the dependency here?

Reply:
Tag message is used for skipping or not skipping the gatk tests which @mlathara asked me to add. Dependency explicitly added to so we can use output from that step to pass tag message to test step.